src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -122,6 +122,14 @@ function BulkEmailForm(props) {
   const onRecipientChange = (event) => {
     if (event.target.checked) {
       dispatch(addRecipient(event.target.value));
+      // if "All Learners" is checked then we want to remove any cohorts, verified learners, and audit learners
+      if (event.target.value === 'learners') {
+        editor.emailRecipients.forEach(recipient => {
+          if (/^cohort/.test(recipient) || /^track/.test(recipient)) {
```

Comment:
For my edification, what is .test() doing? 

My assumption  -- It looks like we have a regex and then the .test is checking if the value of `recipient` matches the regex?

Reply:
Yup exactly and the `^` is telling it to look specifically at the beginning of the `recipient` string